domain-get-legacy/function.json

```diff
@@ -0,0 +1,18 @@
+{
+  "bindings": [
+    {
+      "route": "domains/get/{id}",
+      "authLevel": "anonymous",
+      "type": "httpTrigger",
+      "direction": "in",
+      "name": "req",
+      "methods": ["post"]
```

Comment:
For passivity, make all legacy declarations ["get", "post"], if no `options` object is passed these legacy APIs can be called via GET.

Reply:
Ahh thats right, forget that got changed in PR.